specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -782,6 +904,53 @@ model ConnectedAgentDetails {
   description: string;
 }
 
+/**
+ * The details of the browser automation tool.
+ */
+@added(Versions.v2025_05_15_preview)
+model BrowserAutomationDetails {
+  @added(Versions.v2025_05_15_preview)
+  @doc("Information about the remote browser to be used for automation.")
+  @maxItems(1)
+  @encodedName("application/json", "connection")
+  connection: BrowserAutomationRemoteBrowserConnection;
+}
+
+@added(Versions.v2025_05_15_preview)
+model BrowserAutomationRemoteBrowserConnection {
+  @added(Versions.v2025_05_15_preview)
+  @doc("The connection ID for the remote browser.")
+  @maxItems(1)
+  @encodedName("application/json", "id")
+  connectionId: string;
+}
+
+/**
+ * The details of the Deep Research tool.
+ */
+@added(Versions.v2025_05_15_preview)
+model DeepResearchDetails {
+  @added(Versions.v2025_05_15_preview)
+  @doc("The reasoning model deployment name.")
+  @maxItems(1)
+  @encodedName("application/json", "deployment_name")
+  deploymentName: string;
```

Comment:
Also, based on Sanjeev's comment: "Darren- consider the got-4o as the admin for the DR model. 4o does any preliminary tasks like asking clarifying questions before handing off the research ask to the deep research model. This may change in the future but this is the starting pair team."  -- If this 2nd model may not be needed in the future, perhaps make this field (deploymentName) optional?